src/web/src/components/ScheduleEvent.vue

```diff
@@ -15,6 +15,15 @@
     </div>
   </div>
 </template>
+<script>
+export default {
+  getExploreCourseLink(courseName) {
+    var subject = courseName.split(' ')[0];
+    var courseNumber = courseName.split(' ')[1];
+    return "/explore/" + subject + "/" + subject + "-" + courseNumber;
```

Comment:
one more nit :)
Let use template strings (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to generate the url, makes the template a tad easier to read.

Soemthing like
```js
return `/explore/${subject}/${subject}-${courseNumber}`
```

Besides that lgtm!